src/types/strings/bit.rs

```diff
@@ -16,7 +16,28 @@ use crate::prelude::*;
 /// ```
 pub type BitString = bitvec::vec::BitVec<u8, bitvec::order::Msb0>;
 ///  A fixed length `BIT STRING` type.
+///
+/// IMPORTANT: While N describes the number of bits, also the internal size is for N amount of bytes.
+/// When accessing the bits, use &array[..N] to get the correct  bits, for example.
+/// Also when setting the bits, they should start from the "left". See example.
```

Comment:
I think the better way to describe it is that it expects "big endian" style encoding.